### PR TITLE
Merge core release 2.11.0

### PR DESCRIPTION
**Pull request by makramkd**
## Motivation
We want to stay up to date.

## Solution
Merge 2.11

### CODE REVIEW

File: VERSION
Comment (makramkd) [inline VERSION:1]: Should this file stick around?

Comment (mateusz-sekara) [reply VERSION:1]: We should follow how the Core does it. Worth checking how it's done in Core right now

**Review by mateusz-sekara — APPROVED** [review]

File: VERSION
Comment (makramkd) [reply VERSION:1]: I don't see a VERSION file on their end

Comment (mateusz-sekara) [reply VERSION:1]: Worth trackick the PR that removed this file to understand how it works right now ;)

Comment (mateusz-sekara) [reply VERSION:1]: Worth tracking the PR that removed this file to understand how it works right now ;)

**Review by AnieeG — APPROVED** [review]

**Review by RensR — APPROVED** [review]

**Review by krehermann — APPROVED** [review]

**Review by krehermann — APPROVED** [review]

**Pull request merged**